README.md
# SgtGigdi-Launcher-Roblox
This is the roblox server executor for ScriptHub Key And Much More.


news/News
                                                                                                              
  ,----..                                            .--.--.               ,--,                       ___     
 /   /   \    ,--,                   ,---,  ,--,    /  /    '. ,-.----.  ,--.'|              ,--,   ,--.'|_   
|   :     : ,--.'|                 ,---.'|,--.'|   |  :  /`. / \    /  \ |  | :     ,---.  ,--.'|   |  | :,'  
.   |  ;. / |  |,     ,----._,.    |   | :|  |,    ;  |  |--`  |   :    |:  : '    '   ,'\ |  |,    :  : ' :  
.   ; /--`  `--'_    /   /  ' /    |   | |`--'_    |  :  ;_    |   | .\ :|  ' |   /   /   |`--'_  .;__,'  /   
;   | ;  __ ,' ,'|  |   :     |  ,--.__| |,' ,'|    \  \    `. .   : |: |'  | |  .   ; ,. :,' ,'| |  |   |    
|   : |.' .''  | |  |   | .\  . /   ,'   |'  | |     `----.   \|   |  \ :|  | :  '   | |: :'  | | :__,'| :    
.   | '_.' :|  | :  .   ; ';  |.   '  /  ||  | :     __ \  \  ||   : .  |'  : |__'   | .; :|  | :   '  : |__  
'   ; : \  |'  : |__'   .   . |'   ; |:  |'  : |__  /  /`--'  /:     |`-'|  | '.'|   :    |'  : |__ |  | '.'| 
'   | '/  .'|  | '.'|`---`-'| ||   | '/  '|  | '.'|'--'.     / :   : :   ;  :    ;\   \  / |  | '.'|;  :    ; 
|   :    /  ;  :    ;.'__/\_: ||   :    :|;  :    ;  `--'---'  |   | :   |  ,   /  `----'  ;  :    ;|  ,   /  
 \   \ .'   |  ,   / |   :    : \   \  /  |  ,   /             `---'.|    ---`-'           |  ,   /  ---`-'   
  `---`      ---`-'   \   \  /   `----'    ---`-'                `---`                      ---`-'            
                       `--`-'                                                                                 
NEWS: Welcome To GigdiSploit. You Can Attach At The Right And Execute A Script On The Left. Enjoy!


Version/Download/html/getkey/index.php
<?
    ini_set('display_errors', 'Off');
 error_reporting(0);
    session_start();

    $_SESSION['time'] = time();

    header("Refresh: 10; URL=https://direct-link.net/144159/GigdiSploitKeystep1");
?>


<html>

<meta http-equiv="content-type" content="text/html;charset=UTF-8" />
<head>
<meta charset="utf-8">


<title>GigdiSploit Key System</title>
<meta charset="utf-8" />
<link rel="stylesheet" type="text/css" href="MainDesign.css"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700" rel="stylesheet">

<div class="keysyscontainer">
    <div class="box">
        <span class="title">GigdiSploit Key System</span>
        <span class="text"> You will have to go through four linkvertises. Please do not refresh this page. Redirecting in 10 seconds... </span>
        <span class="text"> No one likes advertisments, but going through this supports the creators of KeySystem. </span>
</html>


Version/Download/html/getkey/step2.php
<?
ini_set('display_errors', 'Off');
 error_reporting(0);
session_start();

if(time() - $_SESSION['time'] < 20 || !isset($_SESSION['time']))
{
	echo "you did not complete the linkvertise";
	echo "<meta http-equiv='Refresh' Content='4;url=lol'>"; 
	return;
}

$_SESSION['timetwo'] = time();

header("Refresh: 5; URL=https://direct-link.net/144159/GigdiSploitKeystep2");  
?>

<html>

<meta http-equiv="content-type" content="text/html;charset=UTF-8" />
<head>
<meta charset="utf-8">


<title>GigdiSploit Key System</title>
<meta charset="utf-8" />
<link rel="stylesheet" type="text/css" href="MainDesign.css"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700" rel="stylesheet">

<div class="keysyscontainer">
    <div class="box">
        <span class="title">GigdiSploit Key System | Step 2</span>
        <span class="text"> You have completed one linkvertise, three more to go! Redirecting in 5 seconds... </span>
        <span class="text"> No one likes advertisments, but going through this supports the creators of GigdiSploit. </span>
</html>


Version/Download/html/getkey/step3.php
<?
ini_set('display_errors', 'Off');
 error_reporting(0);
session_start();

if(time() - $_SESSION['timetwo'] < 20 || !isset($_SESSION['timetwo']))
{
	echo "you did not complete the linkvertise";
	echo "<meta http-equiv='Refresh' Content='4;url=http://sgtgigdi.rf.gd/'>"; 
	return;
}

$_SESSION['timethree'] = time();

header("Refresh: 5; URL=https://direct-link.net/144159/GigdiSploitKeystep3");  
?>

<html>

<meta http-equiv="content-type" content="text/html;charset=UTF-8" />
<head>
<meta charset="utf-8">


<title>GigdiSploit</title>
<meta charset="utf-8" />
<link rel="stylesheet" type="text/css" href="MainDesign.css"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700" rel="stylesheet">

<div class="keysyscontainer">
    <div class="box">
        <span class="title">GigdiSploit | Step 3</span>
        <span class="text"> You have completed two linkvertise, two more to go! Redirecting in 5 seconds... </span>
        <span class="text"> No one likes advertisments, but going through this supports the creators of GigdiSploit. </span>
</html>


Version/Download/html/getkey/step4.php
<?
ini_set('display_errors', 'Off');
 error_reporting(0);
session_start();

if(time() - $_SESSION['timethree'] < 20 || !isset($_SESSION['timethree']))
{
	echo "you did not complete the linkvertise";
	echo "<meta http-equiv='Refresh' Content='4;url=http://sgtgigdi.rf.gd/'>"; 
	return;
}

$_SESSION['timefour'] = time();

header("Refresh: 5; URL=https://direct-link.net/144159/GigdiSploitKeyFinal");  
?>

<html>

<meta http-equiv="content-type" content="text/html;charset=UTF-8" />
<head>
<meta charset="utf-8">


<title>GigdiSploit</title>
<meta charset="utf-8" />
<link rel="stylesheet" type="text/css" href="MainDesign.css"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700" rel="stylesheet">

<div class="keysyscontainer">
    <div class="box">
        <span class="title">GigdiSploit | Step 4</span>
        <span class="text"> You have completed three linkvertise, one more to go! Redirecting in 5 seconds... </span>
        <span class="text"> No one likes advertisments, but going through this supports the creators of GigdiSploit. </span>
</html>


Version/Download/html/getkey/final.php
<?
ini_set('display_errors', 'Off');
error_reporting(0);
session_start();

if(time() - $_SESSION['timefour'] < 20 || !isset($_SESSION['timefour']))
{
	echo "you did not complete the linkvertise";
	echo "<meta http-equiv='Refresh' Content='4;url=http://sgtgigdi.rf.gd/'>"; 
	return;
}

$key = "G7R0WZ-50D59C-VID1C6-N29R1A-0738IP-NALDNS";

$_SESSION['time'] = NULL;
$_SESSION['timetwo'] = NULL;
$_SESSION['timethree'] = NULL;
$_SESSION['timefour'] = NULL;
?>

<html>

<meta http-equiv="content-type" content="text/html;charset=UTF-8" />
<head>
    <script>
        function copyToClipboard() {
            var e = "<?php echo $key ?>";
            var tempItem = document.createElement('input');
            tempItem.setAttribute('type','text');
            tempItem.setAttribute('display','none');
            let content = e;
            if (e instanceof HTMLElement) {
            		content = e.innerHTML;
            }
            tempItem.setAttribute('value',content);
            document.body.appendChild(tempItem);
            tempItem.select();
            document.execCommand('Copy');
            tempItem.parentElement.removeChild(tempItem);
            alert("Successfully copied the key to your clipboard!");
        }
</script>
    
<title>GigdiSploit</title>
<meta charset="utf-8" />
<link rel="stylesheet" type="text/css" href="MainDesign.css" />
<link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700" rel="stylesheet">
<div class="keysyscontainer">
    <div class="box">
        <span class="title">GigdiSploit Key</span>
        <span class="text"> This is your GigdiSploit key. Please do not refresh this page </span>
        <span class="text"> Your key: <?php echo $key ?></span>
        <button onclick = "return copyToClipboard()" class="button Copy">Copy Key</button>
</html>
